src/libical/icaltz-util.c

```diff
@@ -790,7 +790,8 @@ icalcomponent *icaltzutil_fetch_timezone(const char *location)
             // Check if the zone name or either of the offsets have changed
             if (types[prev_idx].gmtoff != zone->gmtoff_from ||
                 types[idx].gmtoff      != zone->gmtoff_to   ||
-                strcmp(types[idx].zname, zone->name)) {
+                (types[idx].zname != NULL &&
+                 strcmp(types[idx].zname, zone->name))) {
```

Comment:
Can zname actually be null, or is this just a clang complaint?

Reply:
clang-analyzer says "/home/allen/projects/libical/libical/src/libical/icaltz-util.c:794:18: warning: Null pointer passed to 1st parameter expecting 'nonnull'
I didn't read the code to see if that condition is possible.